src/dmd/dmodule.d

```diff
@@ -1565,6 +1564,16 @@ extern (C++) final class Module : Package
             buf.prependstring(package_.ident.toChars());
         }
     }
+
+    /** Lazily initializes and returns the escape table.
+    Turns out it eats a lot of memory.
+    */
+    extern(D) Escape* escapetable()
```

Comment:
Curiously, this same code appears in `parseEscapes()` in doc.d. I also wonder why the memory allocated for that needs to be retained and is not recycled.